integration/spark/src/test/scala/org/apache/carbondata/geo/GeoTest.scala

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.carbondata.geo
 
+import scala.collection.mutable
+
```

Comment:
please add some more udf test for intersection status. 
like full inclusion, semi inclusion

Reply:
added more test condition, and checked the test data